compiler/gentree.cpp

```diff
@@ -656,7 +656,7 @@ VertexPtr GenTree::get_expr_top(bool was_arrow, const PhpDocComment *phpdoc) {
       return_flag = false; // array is valid postfix expression operand
       break;
     case tok_tuple:
-      res = get_func_call<op_tuple, op_err>();
+      res = get_func_call<op_tuple, op_none>();
```

Comment:
I am not sure, now such code will now be evaluated: `tuple(1,,,,2)`. Previously, it was an error. Can you check it now?

Reply:
Yes, I have added an appropriate test